plugins/sinks/gcs/README.md

```diff
@@ -0,0 +1,41 @@
+# GCS
+
+Sinks json data to a file in `ndjson` format in a Google Cloud Storage bucket
```

Comment:
Maybe we can rephrase this as:

Sinks json data to a file as [ndjson](http://ndjson.org/) format in Google Cloud Storage bucket